pkgs/by-name/ti/tic-80/package.nix

```diff
@@ -0,0 +1,143 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, libGL
+, alsa-lib
+, libX11
+, xorgproto
+, libICE
+, libXi
+, libXScrnSaver
+, libXcursor
+, libXinerama
+, libXext
+, libXxf86vm
+, libXrandr
+, libxkbcommon
+, wayland
+, wayland-protocols
+, wayland-scanner
+, dbus
+, udev
+, libdecor
+, pipewire
+, libpulseaudio
+, libiconv
+, ...
+}:
+with pkgs;
+stdenv.mkDerivation rec {
+  pname = "tic-80";
+  version = "1.1.2837";
+
+  src = fetchFromGitHub {
+    owner = "nesbox";
+    repo = "TIC-80";
+    rev = "v" + version;
+    sha256 = "sha256-2hSaLWw57F19tSfgJvBgbqW52vCu8p/TmoybYzQEybE=";
+    fetchSubmodules = true;
+    leaveDotGit = true; # Required for TIC-80's version detection via CMake.
```

Comment:
```suggestion
```
we should mock that with fake data because .git is not very reproducible especially witzh submodules

Reply:
Seems like it. I went with your suggestion (which was also @winny-'s initial implementation).